### DISCUSSION TITLE
Post migration script?

### DISCUSSION THREAD

**Discussion by AviBueno** [prisma/prisma#3470] [Q&A] [answered]
Hi, couldn't find a discussion on the subject so I'd like to ask:
Is there a way to execute a post-migration script as part of the migration process?

For example, I have a JSON field with several data fields.
I'd like to create a dedicated column for one of them.
For that I'll add this new column to the schema and create a migration script.
My question is if I could then somehow integrate additional code into the migration script, which will run over the table's rows after a successful migration, pull the specific field's value from the JSON data and populate the newly created column?

If that's not possible then is there some other good alternative to achieve this goal?

Thank you.

**Comment by ryands17** [comment, marked as answer]
Hey @AviBueno 👋 
Currently the only way to do it would be running a script post migration to seed the data. There is a feature request [here](https://github.com/prisma/migrate/issues/492) that tracks the same.

**Comment by Softjey** [comment]
Hi everyone! 👋

I’ve developed a small library to help address this problem: [Prisma Data Migrations](https://www.npmjs.com/package/prisma-data-migrations?activeTab=readme). This library enhances Prisma’s migration system by enabling the use of post-migration scripts for data migrations alongside schema changes.

You can find the repository here: [GitHub Repository](https://github.com/Softjey/prisma-data-migrations).

Feel free to open issues, report bugs, or suggest improvements. I’m also open to pull requests if you’d like to contribute to the project!

Looking forward to your feedback! 🚀

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
